servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/DefaultMultiAddressUrlHttpClientBuilderTest.java

```diff
@@ -54,8 +55,9 @@ void buildWithDefaults() throws Exception {
     @Test
     void buildAggregatedWithDefaults() throws Exception {
         HttpRequester newAggregatedRequester = HttpClients.forMultiAddressUrl()
-                .ioExecutor(CTX.ioExecutor())
-                .executionStrategy(defaultStrategy(CTX.executor()))
+                .initializer((scheme, address, builder) -> builder
+                        .ioExecutor(CTX.ioExecutor())
```

Comment:
Hm... this becomes interesting. Because there is no single `IoExecutor` / `ExecutionStrategy` / `BufferAllocator` for all internal clients anymore, not sure what resulting `HttpClient#executionContext()` should return for multi-address client. We cannot return `UnsupportedOperationException` because `executionContext()` is frequently used to access `BufferAllocator`. @Scottmitch wdyt?